### ISSUE TITLE
Missing dense point cloud

### ISSUE THREAD

**Issue by Vynikal** [openphotogrammetry/meshroomcl#26]
Hello, in 0.8.0 version, I am unable to locate the densePointCloud.abc file, as seen in the node MultiviewStereoCL and the command window. Is it a bug, or am I missing something?
My objective is to export colored point cloud, but I can't find how.

**Comment by revisionarian** [comment]
Hi @Vynikal, the "densePointCloud.abc" file will only be created if you enable the "Compute Visibility" option in the MultiviewStereoCL attributes (it is turned off by default).

If you just want a file with your reconstructed color point cloud, then I'd recommend using the file named "fused.ply" which is always created and is located in the same folder (`.../MeshroomCache/MultiviewStereoCL/xxx/fused.ply`). Creation of the "densePointCloud.abc" file is disabled by default because it requires extra time and RAM to generate, since it contains visibility data in addition to the point cloud data.

**Comment by Vynikal** [comment]
Yes, it took several hours and whole RAM. Then it froze and BSoD'd 😄 
The resulting file is 6GB big JSON. Don't know why or what to do with it.

**Comment by revisionarian** [comment]
Yes, I'd recommend that you just ignore the "densePointCloud.abc" and big JSON files.

Does the "fused.ply" point cloud file work for you?

**Comment by Vynikal** [comment]
Yes, it's ok, CloudCompare is able to import it.
